cmd/aro/mirror.go

```diff
@@ -155,6 +155,17 @@ func mirror(ctx context.Context, log *logrus.Entry) error {
 		}
 	}
 
+	for _, ref := range []string{
+		"quay.io/app-sre/managed-upgrade-operator:v0.1.827-8d67cc6",
+	} {
+		log.Printf("mirroring %s -> %s", ref, pkgmirror.Dest(dstAcr+acrDomainSuffix, ref))
```

Comment:
Since we're reusing lines 161-165 from the `registry.redhat.io` mirror above with the only difference being `srcAuth`, I think we can make a separate function for the `pkgmirror.Copy` stuff. What do you think?

Reply:
With this particular code I would be fine with copy & paste. Having function will require jumping up and down. Moreover this code will potentially go away and be replaced with `az acr import`. 

Would that work for you Caden?